dali/test/python/experimental_mode/test_rn50_pipeline.py

```diff
@@ -1,23 +1,30 @@
 import nvidia.dali.experimental.dynamic as ndd
-import nvidia.dali.experimental.dynamic.ops as ops
 import nvidia.dali as dali
 import nvidia.dali.fn as fn
 import os
 import numpy as np
 from test_utils import get_dali_extra_path
 
+file_root = os.path.join(get_dali_extra_path(), "db/single/jpeg")
+img_list = os.path.join(file_root, "image_list.txt")
+batch_size = 16
+
+rng1 = ndd.random.rng(seed=42)
+rng2 = ndd.random.rng(seed=43)
+rng3 = ndd.random.rng(seed=44)
+
 
 def get_rn50_pipeline_fn(max_batch_size):
```

Comment:
We could get rid of this extra layer of functions. To make the usage more realistic, I'd recommend using only one RNG here and passing it to the function.